src/main.js

```diff
@@ -18,31 +17,22 @@ var titleInput = document.querySelector('.user-title');
 var firstDescriptorInput = document.querySelector('.user-desc1');
 var secondDescriptorInput = document.querySelector('.user-desc2');
 
-
-
-// We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 
-var currentCover// = savedCovers[(savedCovers.length)]
-// Add your event listeners here 👇
+var currentCover
 randomCoverButton.addEventListener('click', randomCover);
 makeNewCover.addEventListener('click', changeToMakeNewCover);
 viewSavedCoversButton.addEventListener('click', changeToSavedCovers);
 homeButton.addEventListener('click', goHome)
 makeMyBook.addEventListener('click', makeBookCover)
 
-
-// Create your event handlers and other functions here 👇
-
-
-// We've provided one function to get you started
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-function currentCover(array) {
+function makeCurrentCover(array) {
```

Comment:
If we edit the array below change this one

Reply:
Done